tests/CMakeLists.txt

```diff
@@ -60,3 +60,40 @@ foreach(test_lang c-sharp c cpp d java pawn objective-c vala ecma imported)
     endif()
   endforeach()
 endforeach()
+
+if(PythonInterp_FOUND)
+  if (CMAKE_CONFIGURATION_TYPES)
+    # Not really, but just to make the following if() true
+    set(_build_type release)
+    set(_configs CONFIGURATIONS Release RelWithDebInfo MinSizeRel)
```

Comment:
Not entirely sure what the difference between `_build_type` and `_configs` is and how they are used exactly.

Reply:
(Sorry for the delay; wrote this yesterday and managed to never post it :flushed:...)

`_build_type` is the current build type. (Except, *here*, not really; per the comment, we're setting it to "release" even though that is sort of a lie, because we want the next `if()` to be true. In the `else()`, it's the actual configuration type.) `_configs` is a set of *additional parameters* passed to `add_test` so that the test will only be included if the configuration given to CTest (`ctest -c <config>`) is a release configuration.

Hope that helps!